util/a2e/atoe.c

```diff
@@ -45,6 +45,9 @@
  * Include all system header files.
  * ======================================================================
  */
+#if defined(__open_xl__)
+#define _EXT
+#endif
```

Comment:
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif
```suggestion
#endif /* defined(__open_xl__) */
```

Reply:
Done